### PR TITLE
put in info for current officers

### PR DESCRIPTION
**Pull request by bghsstemclub**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -143,23 +147,30 @@ <h1><b>WHAT WE DO</b></h1>
...
+                  <div class="fade">
+                    <div class="center3">Code B</div>
+                  </div>
+                </a>
               </div>
-            </a>
+            </div>
           </div>
         </div>
       </div>
```
Comment (Teamustu) [inline about.html:175]: This looks like an extra end div that has no opening tag.

File: about.html [hunk 2]
```diff
@@ -143,23 +147,30 @@ <h1><b>WHAT WE DO</b></h1>
   <!--box consists of an image with hyperlink-->
   <div class="box0" style="background-image: linear-gradient(to bottom right, rgb(103, 222, 226),rgb(39, 146, 233));">
     <div class="row">
-      <div class="column2" style="padding: 20px 50px;">
-        <h1 style="font-size: 64px"><b>BIGGER ASSOCIATION</b></h1>
+      <div class="column2">
+        <h1><b>BIGGER ASSOCIATION</b></h1>
+        <p>
+          Every year, STEM provides students the opportunities to expand their knowledge and compete through a variety of projects, and competitions from different organizations.
+          We have two main competitions: MESA and Sideline, both of which require students to design, build and work in a team to make a real-life pratical device.
```
Comment (Teamustu) [inline about.html]: Misspelled practical.

File: about.html [hunk 3]
```diff
@@ -126,13 +126,17 @@ <h1><b>OUR VISION</b></h1>
...
             </div>
           </a>
         </div>
       </div>
       <div class="column2">
-      <h1><b>WHAT WE DO</b></h1>
+        <h1><b>WHAT WE DO</b></h1>
+        <p>
+          Bolsa Grande STEM strives to offer students the chance to learn more about and get hands-on experience in the STEM fields, 
+          be it by constructing engineering projects, coding complex algorithm, or participating in exhilirating and meaningful icebreakers -
```
Comment (Teamustu) [inline about.html]: Misspelled exhilarating.

**Review by Teamustu — CHANGES_REQUESTED** [review]
Looks good besides some minor issues.

**Review by khoanguyendinh — APPROVED** [review]

**Pull request merged**